src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -623,14 +698,39 @@ public bool HasSkipLocalsInitAttribute
         internal bool IsAutoPropertyWithGetAccessor
             => IsAutoProperty && _getMethod is object;
 
-        protected bool IsAutoProperty
-            => (_propertyFlags & Flags.IsAutoProperty) != 0;
+        internal bool IsAutoProperty
+        {
+            get
+            {
+                return (_propertyFlags & Flags.IsAutoProperty) != 0 || BackingField is { IsCreatedForFieldKeyword: true };
```

Comment:
>BackingField [](http://example.com/codeflow?start=71&length=12)

I do not think this makes things any better. We are trading one way to bind accessors with another.